apps/voting-disputable/test/contracts/voting_execution.js

```diff
@@ -22,98 +22,168 @@ contract('Voting', ([_, owner, holder20, holder29, holder51]) => {
     await token.generateTokens(holder20, bigExp(20, 18))
   })
 
-  beforeEach('deploy voting', async () => {
-    voting = await deployer.deployAndInitialize({ owner, supportRequired: REQUIRED_SUPPORT, minimumAcceptanceQuorum: MINIMUM_ACCEPTANCE_QUORUM, voteDuration: VOTE_DURATION, overruleWindow: OVERRULE_WINDOW })
-  })
-
   describe('execute', () => {
-    let voteId, executionTarget, script
     const from = holder51
+    let voteId, executionTarget, script
 
-    context('with an empty script', () => {
-      beforeEach('create script', async () => {
-        ({ script } = await voteScript(0))
-      })
-
-      it('cannot be executed', async () => {
-        ({ voteId } = await createVote({ voting, script, from }))
+    context('without delayed execution', () => {
```

Comment:
Seems like we could share a lot of the tests between the case of delayed execution = 0 and the one > 0.